docs/proposals/updating_upstream_images.md

```diff
@@ -0,0 +1,54 @@
+Updating upstream image tags for supported branches
+---------------------------------------------------
+Our logging team received a few issue and bug reports for a stable upstream 
+branch, where the proper code was already merged in github, but propagating
+the code to upstream images was out of sync with openshift ansible releases.
+
+Upstream image tags
+-------------------
+For upstream images, we have the `latest` image tag and then image tag based
+on the `git tag` (for example `v3.6.1`). The `latest` is frequently updated
+when we merge a PR to the `master` branch, the `v3.6.1` tag is created once and 
+does not get updated. For downstream images, we also have updating tags based
+on `git branch` (for example branch `rhaos-3.6-rhel-7` to image tag `v3.6`) 
+and these get updated by our release engineers.
+
+I would like to propose similar updating tags for the upstream images based on 
+the `git branch`.
+
```

Comment:
To clarify the table below you could add something like the following here:
"We will create a floating tag that matches the pattern `vX.Y` that will be updated whenever we push to its corresponding branch or when a `vX.Y.Z` tag is created. For example, if we push an update to `release-3.6` then we would also rebuild and pushing a update to the tag `v3.6`. If we create a github tag `v3.6.5` then we would push a new image tagged `v3.6.5` and also point `v.3.6` to this newly created image tag since it should contain the latest 3.6 change."

Reply:
Thanks for the review, added very similar text to what you recommended and modified the table so its easier to read.